Userland/Libraries/LibUnicode/CharacterTypes.cpp

```diff
@@ -401,4 +401,13 @@ BidiClass bidirectional_class(u32 code_point)
     return char_direction_to_bidi_class(direction);
 }
 
+bool should_continue_beyond_word(Utf8View const& word)
```

Comment:
I meant to put this on the `InputEventsTarget` base class, but this is actually fine too. But if we want it in LibUnicode, I think:

* It belongs more in Segmenter.h/cpp
* The name `should_continue_beyond_word` could be improved because on its own, it doesn't make as much sense without the immediate context of its usage in the same class. Or, we can just make it a static method on the Segmenter class, then call sites of `Segmenter::should_continue_beyond_word` makes enough sense IMO.

(You also don't need the `Unicode::` prefixes below in this file).